Engineering-Reference/M365SourceAO.md

```diff
@@ -43,64 +43,59 @@ Default Value: Off
 
 SharePoint Migration API is recommended for all migrations to SharePoint Online and OneDrive for Business. For more information see the article here.
 
-Change Conditions: ??
+Change Conditions: Turn on to use the SharePoint Migration API
 
 ### SharePoint Admin Url <a name="sharepointadmin"></a>
 {: .no_toc }
 Default Value: None
 
 The Url for the SharePoint admin center e.g. https://tenant-admin.sharepoint.com
 
-Change Conditions: ??
+Change Conditions: Enter the URL for the SharePoint admin center
 
 ### Timeout <a name="sharepointtime"></a>
 {: .no_toc }
 Default Value: 1800000
 
 The timeout that will apply to communications with the SharePoint server.
 
-Change Conditions: ??
+Change Conditions: Value can be changed. Can cause performance issues, not recommended.
 
 ### Include Classic Team Sites in User List <a name="includeclassicteam"></a>
 {: .no_toc }
 Default Value: Off
 
 Specifies whether Team Sites will be obtained when getting a user list.
 
-Change Conditions: ??
+Change Conditions: When turned on, Sharepoint sites will be listed in the items list when getting items from source.
 
 ### Retry Count <a name="retrycount"></a>
 {: .no_toc }
 Default Value: 10
 
 The number of times an operation will be attempted before failing.
 
-Change Conditions: ??
+Change Conditions: Value can be changed. Can cause performance issues, not recommended.
 
 ### Document Sharing/Permissions <a name="docsharingoperm"></a>
 {: .no_toc }
 Default Value: Share Documents
 
 Choose whether the document should be shared as per the source file ACLs if they can be resolved to email addresses (see documentation).
 
-Change Conditions: ??
+Change Conditions: Select 'No Sharing' to prevent the migration of file/folder permissions or 'Share Documents and Send Notifications' to migrate permissions *and* send email notifications to recipients.
```

Comment:
Maybe . Turning this off will have the tool only migrate owner level permissions.

Reply:
done